src/http_servers/html-server.js

```diff
@@ -0,0 +1,46 @@
+const { readFileSync } = require('fs');
+const { Readable } = require('stream');   
+const { join } = require('path');
+
+const TEMPLATE_PATH = 'html/index.html';
+const customData = {
+    message: 'A custom message from html-server',
+};
+
+
+require('http')
+  .createServer((req, res) => {
+      res.writeHead(200, {'Content-Type': 'text/html'});
+      createReadableStream().pipe(res);
+    })
+  .listen(3002);  
+
+
+  function createReadableStream() {
+    const readStream = new Readable({
+      read() {
+        const htmlTemplate = readFileSync(join(__dirname, TEMPLATE_PATH), 'utf8');
```

Comment:
`fs` has its method `fs.createReadStream()`, the idea of hometask was to implement both approaches to send html content - with `fs.readFileSync()` and with `fs.createReadStream()` 
`fs.createReadStream()` will return the stream object, then you can pipe replace method to change content, and at the end pipe it to a response stream.

Reply:
My bad, I understood it wrong. I will update it accordingly.

Reply:
Updated the code to implement two approaches `fs.readFileSync` and `fs.createReadStream`